tools/tkn-results/cmd/records_list.go

```diff
@@ -35,9 +29,12 @@ var (
 		},
 		Args: cobra.ExactArgs(1),
 	}
-)
+
+	listFlags(cmd.Flags())
+
+	return cmd
+}
 
 func init() {
-	listFlags(recordsListCmd.Flags())
-	recordsCmd.AddCommand(recordsListCmd)
+	recordsCmd.AddCommand(listRecordsCommand())
```

Comment:
nit: Ideally we wouldn't even have this `init` and we would add the command on usage, like [here](https://github.com/tektoncd/cli/blob/main/pkg/cmd/root.go#L85) but that can be done in a follow-up 😉

Reply:
I'll fix this straight away, thanks!